compiler/tools/fpp-to-cpp/test/component/base/ActiveAsyncProductsComponentAc.ref.hpp

```diff
@@ -7,7 +7,7 @@
 #ifndef ActiveAsyncProductsComponentAc_HPP
 #define ActiveAsyncProductsComponentAc_HPP
 
-#include <FpConfig.hpp>
+#include <Fw/FPrimeBasicTypes.hpp>
```

Comment:
Let's write `#include "Fw/FPrimeBasicTypes.hpp"` instead of `#include <Fw/FPrimeBasicTypes.hpp>`, as elsewhere.

Reply:
Agreed!